sdk/iotcentral/arm-iotcentral/package.json

```diff
@@ -20,13 +21,13 @@
   "module": "./esm/iotCentralClient.js",
   "types": "./esm/iotCentralClient.d.ts",
   "devDependencies": {
-    "typescript": "^3.5.3",
+    "typescript": "^3.6.0",
     "rollup": "^1.18.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-sourcemaps": "^0.4.2",
     "uglify-js": "^3.6.0"
   },
-  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/iotcentral/arm-iotcentral",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js",
```

Comment:
Was this change intentional? It is not related to the Identity story we are working on at the moment

Reply:
Perhaps it was due to recent work in code gen? cc @sarangan12 @joheredi

Reply:
Shouldnt be... #13479 is the most recent auto generated mgmt plane pkg PR and the homepage there points to the right folder

Reply:
@ramya-rao-a I reverted those back by hand for now.

@colawwj and @qiaozha regarding https://github.com/Azure/azure-sdk-for-js/pull/13084, would you please tell me the exact autorest command you use for that generation? I think you may have used a configuration that sets the homepage property correctly.